docs/http.rst

```diff
@@ -487,3 +496,78 @@ Remove a given ChannelID subscription from a UAID.
 **Return Codes:**
 
 See :ref:`errors`.
+
+Report ACKs and NAKs
+~~~~~~~~~~~
```

Comment:
The ~ needs to match the line length above it. Does it match here and the mismatch is just the github font?